rules/Railway.js

```diff
@@ -1,8 +1,5 @@
 "use strict";
 
-var MATCH_OVERLAP_THRESHOLD = 0.75;
```

Comment:
These were only being used by one of the rules js files, so removed them from the rest.